src/common/test/myProfileMocking.ts

```diff
@@ -155,3 +207,189 @@ export const getVerifiedData = (
   },
   ...overrides,
 });
+
+function createNewProfileNode(dataType: EditDataType): MultiItemProfileNode {
+  return {
+    ...formConstants.EMPTY_VALUES[dataType],
+  };
+}
+
+function findNode(
+  nodes: MultiItemProfileNode[],
+  id: string,
+  returnIndex = false
+): number | MultiItemProfileNode | undefined {
+  const func = returnIndex ? 'findIndex' : 'find';
+  return nodes[func](node => node.id === id);
+}
+
+function getEdges(
+  profileData: ProfileData,
+  dataType: EditDataType
+): EdgeList | null {
+  const targetPath = `${dataType}.edges`;
+  return _.get(profileData, targetPath, null);
+}
+
+function getEdgeTypeName(
+  dataType: EditDataType
+): 'EmailNodeEdge' | 'AddressNodeEdge' | 'PhoneNodeEdge' {
+  return dataType === 'emails'
+    ? 'EmailNodeEdge'
+    : dataType === 'phones'
+    ? 'PhoneNodeEdge'
```

Comment:
Personally I find double ternary little confusing, but could be just that I haven't got to used to it. Would probably prefer regular if else for clarity

Reply:
Done.